### PR TITLE
[ADD] l10n_be_iso20022_pain

### PR DESCRIPTION
**Pull request by sbidoul**
This PR depends on https://github.com/OCA/bank-payment/pull/80

### CODE REVIEW

**Comment by lmignon** [comment]
:+1:  LGTM (code review, no tests)

File: l10n_be_sepa/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'SEPA Support for Belgium',
+    'version': '1.0',
+    'author': 'ACSONE SA/NV',
```
Comment (adrienpeiffer) [inline l10n_be_sepa/__openerp__.py]: Should we not add OCA as author ?

File: l10n_be_sepa/README.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+* https://www.febelfin.be/fr/paiements/directives-et-protocoles-standards-bancaires
+* https://www.febelfin.be/nl/betaalverkeer/richtlijnen-en-protocollen-bankstandaarden
+
+[1] https://www.febelfin.be/sites/default/files/Payments/AOS-OGMVCS.pdf
+
+Contributors
+------------
+* Stéphane Bidoul (ACSONE)
+
```
Comment (adrienpeiffer) [inline l10n_be_sepa/README.rst]: Should we not add maintainer section ?

**Comment by luc-demeyer** [comment]
The name of the module is confusing and incorrect. 
SEPA is the Single European Payment Area, hence payments in euro, whereas the pain protocol allows both SEPA and non-SEPA payments (cr. account_pain module which supports both).

A more correct name would be l10n_be_pain.(or l10n_be_iso20022_pain).

I think that this module (once renamed) is also the correct place to add the logic for the  InitgPty.Id.OrgId.Othr.Id and InitgPty.Id.OrgId.Othr.Issr fields (cf. febelfin specs).

I can make a PR for this but prefer to do this on a module with a correct naming.

**Comment by sbidoul** [comment]
@adrienpeiffer I adapted the README.
@luc-demeyer I renamed the module.

Regarding the InitgPty.Id.OrgId.Othr I'll do something when we have reached a conclusion in https://github.com/OCA/bank-payment/pull/131 and/or https://github.com/OCA/bank-payment/pull/112

**Comment by adrienpeiffer** [comment]
:+1: Thanks !

**Comment by lmignon** [comment]
:+1:  (Code review)

**Pull request merged**